bb.edn

```diff
@@ -0,0 +1,27 @@
+{:paths ["scripts/src"]
+ :tasks
+ {dev:watch
+  logseq.tasks.dev/watch
+
+  dev:open-dev-electron-app
+  logseq.tasks.dev/open-dev-electron-app
+
+  -dev:electron-start
+  {:depends [dev:watch dev:open-dev-electron-app]}
+
+  dev:electron-start
+  {:doc "Start electron dev by watching assets and opening dev app"
+   ;; Parallel execution - https://book.babashka.org/#parallel
+   :task (run '-dev:electron-start {:parallel true})}
```

Comment:
This is like run-p in node but much more powerful as tasks that we can parallelize aren't strings but rather a major subset of clojure